### ISSUE TITLE
Sometimes causes NVDA to freeze silently

### ISSUE THREAD

**Issue by cary-rowen** [yplassiard/nvda-translate#16]
In some regions, the Google Translate Engine is not accessible, and attempting to turn on just-in-time translation will cause NVDA to freeze and NVDA must be restarted.
I'm guessing that the wait trying to request Google's servers is blocking NVDA.

**Comment by yplassiard** [comment]
On regions where Google Translate is not supported, the add-on should actually turn itself off. I'll see if I can do something to prevent the whole NVDA freeze, but it would require some not so straightforward changes.
